src/components/control_bar/_control_bar.scss

```
@@ -0,0 +1,222 @@
+  width: 100%;
+  color: $controlBarText;
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  // This large box shadow helps prevent a flicker of dark
+  // background when the content is shown and hidden
+  box-shadow: inset 0 $controlBarInitialHeight 0 $controlBarBackground, inset 0 600rem 0 $euiColorLightestShade;
+  z-index: 20000;
+  left: 0;
+  bottom: 0;
+  transform: translateY(0);
+  box-sizing: border-box;
+
+  &.euiControlBar--navCollapsed {
+    left: $euiNavDrawerWidthCollapsed;
+    width: calc(100% - #{$euiNavDrawerWidthCollapsed});
+  }
+
+  &.euiControlBar--navExpanded {
+    left: $euiNavDrawerWidthExpanded;
+    width: calc(100% - #{$euiNavDrawerWidthExpanded});
+  }
+
+  &.euiControlBar--open {
+    animation-duration: $euiAnimSpeedNormal;
+    animation-timing-function: $euiAnimSlightResistance;
+    animation-fill-mode: forwards;
+  }
+
+  &.euiControlBar--open.euiControlBar--large {
+    animation-name: openPanelLarge;
+    height: calc(100vh - 5rem);
+    bottom: -100vh;
+  }
+
+  &.euiControlBar--open.euiControlBar--medium {
+    animation-name: openPanelMedium;
+    height: 50vh;
+    bottom: -50vh;
+  }
+
+  &.euiControlBar--open.euiControlBar--small {
+    animation-name: openPanelMedium;
+    height: 25vh;
+    bottom: -25vh;
+  }
+
+  &.euiControlBar--large {
+    height: $controlBarInitialHeight;
+  }
+
+  &.euiControlBar--medium {
+    height: $controlBarInitialHeight;
+  }
+
+  &.euiControlBar--small {
+    height: $controlBarInitialHeight;
+  }
+}
+
+.euiControlBar__controls {
+  height: $controlBarInitialHeight;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  overflow-x: auto;
+}
+
+.euiControlBar__content {
+  @include euiScrollBar;
+  overflow-y: scroll;
+  width: 100%;
+  height: calc(100% - #{$controlBarInitialHeight});
+  background-color: $euiColorLightestShade;
+  box-sizing: border-box;
+  animation-name: showContent;
+  animation-duration: $euiAnimSpeedSlow;
+  animation-iteration-count: 1;
+  animation-timing-function: $euiAnimSlightResistance;
+  color: $euiColorDarkestShade;
+}
+
+@keyframes openPanelLarge {
+  0% {
+    // Accounts for the initial height offset from the top
+    transform: translateY(calc((#{$controlBarInitialHeight} * 3) * -1));
+  }
+
+  100% {
+    transform: translateY(-100vh);
+  }
+}
+
+@keyframes openPanelMedium {
+  0% {
+    transform: translateY(-$controlBarInitialHeight);
+  }
+
+  100% {
+    transform: translateY(-50vh);
+  }
+}
+
+@keyframes openPanelSmall {
+  0% {
+    transform: translateY(-$controlBarInitialHeight);
+  }
+
+  100% {
+    transform: translateY(-25vh);
+  }
+}
+
+@keyframes showContent {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.euiControlBar__buttonIcon {
+  min-width: $controlBarInitialHeight;
+}
+
+.euiControlBar__button {
+  border-radius: .125rem;
+  height: $euiSizeXL;
+  margin-left: $euiSizeXS;
+  flex-shrink: 0;
+
+  &:enabled:hover {
+    transform: translateY(0);
```

Easier is just to say none, in case we ever add more transforms 😉 
```suggestion
    transform: none;
```